ci/teams/rhtap-servicerelease/kustomization.yaml

```diff
@@ -7,6 +7,7 @@ resources:
 - release-strategy.yaml
 - persistent-volume-claim.yaml
 - release-pipeline-sa.yaml
+- external-secrets
```

Comment:
https://github.com/redhat-appstudio/infra-deployments/blob/main/components/cluster-secret-store/base/appsre-stonesoup-vault-secret-store.yaml -> the namespace rhtap-servicerelease-tenant should be there

Reply:
fixed.